test/simulator/timestamp/call_log_manager/call_log_manager.h

```diff
@@ -33,7 +33,13 @@
 
 using json = nlohmann::json;
 
-enum class api_method { close_session, open_session, set_timestamp, wiredtiger_open };
+enum class api_method {
+    close_session,
+    open_session,
+    set_timestamp,
+    query_timestamp,
```

Comment:
We could sort them A-Z

Reply:
Updated in [4a62e95](https://github.com/wiredtiger/wiredtiger/pull/8145/commits/4a62e95768f31c15421f190f70c93abf28157f99)